src/main/java/org/fundacionjala/at15/pokemon/commands/create/IncompleteArguments.java

```diff
@@ -0,0 +1,8 @@
+package org.fundacionjala.at15.pokemon.commands.create;
+
+public class IncompleteArguments extends Exception {
+
+    public IncompleteArguments(String message) {
+        super(message);
+    }
+}
```

Comment:
I think you can create a new Package only for Exceptions

Reply:
Sounds great!